src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,28 @@
-export const PostInfo = () => <>Put the post here</>;
+import { UserInfo } from "../UserInfo/UserInfo";
+import { CommentList } from "../CommentList";
+import "./PostInfo.scss";
+
+export const PostInfo = ({ post }) => {
+  const { title, body, user, comments } = post;
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+        <p>
+          {" Posted by  "}
+
+          <UserInfo user={user} />
+        </p>
+      </div>
+      <p className="PostInfo__body">{body}</p>
+      {comments.length
+        ? <CommentList comments={comments} />
+        : (
+          <>
+            <hr />
+            <b data-cy="NoCommentsMessage">No comments yet</b>
+          </>
+        )}
```

Comment:
don't forget about indents
```suggestion
        <h3 className="PostInfo__title">{title}</h3>
        
        <p>
          {" Posted by  "}

          <UserInfo user={user} />
        </p>
      </div>
      
      <p className="PostInfo__body">{body}</p>
      
      {comments.length
        ? <CommentList comments={comments} />
        : (
          <>
            <hr />
            <b data-cy="NoCommentsMessage">No comments yet</b>
          </>
        )}
```